content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -168,212 +165,161 @@ To define the `Config` trait for the proof-of-existence pallet:
 Now that you've configured the pallet to emit events, you are ready to define those events.
 As described in [Design the application](#design-the-application), the proof-of-existence pallet emits an event under the following conditions:
 
-- When a new proof is added to the blockchain.
-- When a proof is revoked.
```

Comment:
changed "proof" to "claim". a hash is not really a "proof", and all the actions in the pallet are like "create claim" and "revoke claim" so just updated all instances of this noun